llvm/lib/Target/X86/X86FlagsCopyLowering.cpp

```diff
@@ -688,7 +707,7 @@ bool X86FlagsCopyLoweringPass::runOnMachineFunction(MachineFunction &MF) {
       }
 #endif
 
-  return true;
+  return !Copies.empty();
```

Comment:
Shouldn't we always return `true` with checking `MI.getOperand(0).getReg() == X86::EFLAGS`?